maps/torch/job/torch_jobs_boh.dm

```diff
@@ -265,7 +265,7 @@
 		/datum/mil_branch/solgov
 	)
 	allowed_ranks = list(
-		/datum/mil_rank/fleet/o1,
+        /datum/mil_rank/fleet/o1,
```

Comment:
This line got no proper indentation anymore, you kinda messed that up a bit, just remove the additional spaces.

Reply:
Loorey.
I didint get it
I didint put the additional spaces and I cant find it
![image](https://user-images.githubusercontent.com/67030229/99881466-c613c400-2bf8-11eb-9457-1fddd318aedb.png)

Reply:
![image](https://user-images.githubusercontent.com/39163886/99881526-aee5ce00-2c1a-11eb-91b6-dc04ce80b0b4.png)
Line 268 in `maps/torch/job/torch_jobs_boh.dm`, check if it's really tabs and not spaces, Code Checks are failing because of it too.

Reply:
ooooh i used space
I used tab now

Reply:
its fixed thank you

Reply:
![image](https://user-images.githubusercontent.com/39163886/99881674-5ebb3b80-2c1b-11eb-9692-a3bde333df80.png)
It's still missing a second tab.

Reply:
okay

Reply:
Sorry its to many details and i still learning

Reply:
But thank you anyway